defaults/main.yml

```diff
@@ -34,7 +35,7 @@ buildkite_agent_expose_secrets: false
 buildkite_agent_builds_dir:
   Darwin: "{{ buildkite_agent_home_dir[ansible_os_family] }}/builds"
   Debian: "{{ buildkite_agent_home_dir[ansible_os_family] }}/builds"
-  Windows: "c:/b"  # long filenames will overflow
+  Windows: c:/b  # long filenames will overflow
```

Comment:
quote usage is inconsistent between the `Windows` entry and the others in this dict.